gc.rb

```diff
@@ -196,6 +196,24 @@ def self.stat hash_or_key = nil
     Primitive.gc_stat hash_or_key
   end
 
+  # :nodoc:
+  # call-seq:
+  #    GC.stat_size_pool(size_pool_idx) -> Hash
+  #    GC.stat_size_pool(size_pool_idx, hash) -> Hash
+  #    GC.stat_size_pool(size_pool_idx, :key) -> Numeric
+  #
+  # Returns a Hash containing information about a size pool in the GC.
+  #
+  # The contents of the hash are implementation specific and may be changed in
+  # the future.
```

Comment:
```suggestion
  # The contents of the hash are implementation specific and may change in
  # the future without notice.
```

Remove unnecessary passive form and be more explicit about stability.

Reply:
Thanks, I just copied from the docs in GC.stat. I also updated that in f8ead3a72f4cde3dbec48d33f8921a97b090f799.